pkgs/games/srb2/default.nix

```diff
@@ -77,8 +77,10 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   patches = [
-    # Fix unknown command "CPMAddPackage" by not using Ccache.cmake
+    # Make the build work without internet connectivity
+    # See: https://build.opensuse.org/request/show/1109889
     ./cmake.patch
```

Comment:
Would it be possible to fetchpatch this?

Reply:
The openSUSE patches use `CMakeLists.txt.orig` etc. so I had to manually create these patches for Nix